v0/destinations/splitio/data/EventConfig.json

```diff
@@ -0,0 +1,78 @@
+[
+  {
+    "destKey": "eventTypeId",
+    "sourceKeys": [
+      "eventTypeId",
+      "properties.eventTypeId",
+      "event",
+      "name",
+      "traits.eventTypeId",
+      "context.traits.eventTypeId"
+    ],
+    "metadata": {
+      "type": "toString"
+    },
+    "required": true
+  },
+  {
+    "destKey": "environmentName",
+    "sourceKeys": [
+      "environmentName",
```

Comment:
environment name is taken from web app why again map here?
in root no random keys are present in rudder payload. please go through rudder spec again.

Reply:
eliminated environment name from here. will take it from config only